src/components/NavBar.vue

```diff
@@ -13,6 +13,7 @@
         is-nav>
         <b-navbar-nav v-if="isLoggedIn">
           <b-nav-item to="/recordings">Recordings</b-nav-item>
+          <b-nav-item to="/analysis">Analysis</b-nav-item>
```

Comment:
I think Analysis can go before Recordings. It'll likely become the thing that most people look at first.